service/conf/routes

```diff
@@ -17,7 +15,6 @@ PATCH	/v1/user/update					@controllers.usermanagement.UserController.updateUserP
 POST	/v1/user/login					@controllers.usermanagement.UserController.login()
 POST	/v1/user/logout					@controllers.usermanagement.UserController.logout()
 POST	/v1/user/changepassword			@controllers.usermanagement.UserController.changePassword()
-GET  	/v1/user/getprofile/:uid			@controllers.usermanagement.UserController.getUserProfile(uid:String)
```

Comment:
Check if portal or mobile is using /v1/user/content/state. Or it is onboarded or not.

Reply:
onboarded  uri is /v1/content/state/read , actually inside routes /v1/user/content/state and /v1/content/state/read is pointing to same Controller method.